test/integration.test.ts

```diff
@@ -485,6 +485,20 @@ const testCases: {
       expect(await existsFile(join(dir, './dist/index.d.ts'))).toBe(true)
     },
   },
+  {
+    name: 'tsconfig-override',
+    dir: 'tsconfig-override',
+    args: ['--tsconfig', 'tsconfig.build.json'],
+    async expected(dir) {
```

Comment:
One way is to test `target`, you can do es5, then you get `var` in the output, and another one is still es8 by default.

We're using the new way to write interagtion test now, you can checkout #484 to see how we do it now. Basically have a new `index.test.ts` under that folder and put test there. so we can run them as a standalone test

Reply:
rewrote the test!